src/vtex-ng-filter.html

```diff
@@ -13,6 +13,8 @@ <h3>
     </button>
   </h3>
 
+  <span ng-if="infoMessage" translate>{{ infoMessage }}</span>
```

Comment:
do you need this translate attribute? Is `infoMessage` a string with the key to be translated or the string of the message already localized?